frontend/src/TableComponents/SortMenu.jsx

```diff
@@ -0,0 +1,143 @@
+import { Button, Menu, MenuItem, Divider } from "@mui/material";
+import { useState } from "react";
+import "./SortMenu.css";
+
+function SortMenu({ setSortType }) {
+  const [anchorEl, setAnchorEl] = useState(null);
+  const open = Boolean(anchorEl);
+  const playerInfoSortOptions = [
+    ["id asc", "ID"],
+    ["player_name asc", "Alphabetical"],
+  ];
+  const scoringSortOptionsDecreasing = [
+    ["PTS desc", "Points Scored"],
+    ["field_percent desc", "FG % Decreasing"],
+    ["three_percent desc", "Three % Decreasing"],
+    ["two_percent desc", "Two % Decreasing"],
+    ["effect_fg_percent desc", "Effective FG % Decreasing"],
+    ["ft_percent desc", "FT% Decreasing"],
+  ];
+
+  const scoringSortOptionsIncreasing = [
+    ["field_percent asc", "FG % Increasing"],
+    ["three_percent asc", "Three % Increasing"],
+    ["two_percent asc", "Two % Increasing"],
+    ["effect_fg_percent asc", "Effective FG % Increasing"],
+    ["ft_percent asc", "FT% Increasing"],
+  ];
+
+  const miscSortOptionsDecreasing = [
+    ["ORB desc", "ORB Descending"],
+    ["DRB desc", "DRB Decreasing"],
+    ["TRB desc", "TRB Decreasing"],
+    ["AST desc", "AST Decreasing"],
+    ["STL desc", "STL Decreasing"],
+    ["BLK desc", "BLK Decreasing"],
+    ["TOV desc", "TOV Decreasing"],
+    ["PF desc", "PF Decreasing"],
+  ];
+
+  const miscSortOptionsIncreasing = [
+    ["ORB desc", "ORB Increasing"],
+    ["DRB desc", "DRB Increasing"],
+    ["TRB desc", "TRB Increasing"],
+    ["AST desc", "AST Increasing"],
+    ["STL desc", "STL Increasing"],
+    ["BLK desc", "BLK Increasing"],
+    ["TOV desc", "TOV Increasing"],
+    ["PF desc", "PF Decreasing"],
+  ];
```

Comment:
Since these constants don't rely on scope, it would be more efficient to define these above of the component using constant naming convention like `const SCORING_SORT_OPTIONS_DECREASING`. Otherwise these would be defined on every re-render of the component